build.cake

```diff
@@ -299,18 +299,14 @@ Task("provision-androidsdk")
             }
         }
 
-        if (!IsRunningOnWindows ()) {
-            if(!String.IsNullOrWhiteSpace(androidSDK))
-            {
-                await Boots (androidSDK);
-            }
-            else
-                await Boots (Product.XamarinAndroid, releaseChannel);
-        }
-        else if(!String.IsNullOrWhiteSpace(androidSDK))
+        if(!String.IsNullOrWhiteSpace(androidSDK))
         {
             await Boots (androidSDK);
         }
+        else
+        {
+            await Boots (Product.XamarinAndroid, releaseChannel);
```

Comment:
Yeah that probably explains why this didn't work. I had to put a variable value in the Azure Pipeline to get the actual right version. This would keep coming up with 12. So that is probably why, I'll try this and see if that fixes it. A bit less magic than the AzDO variable